website/pages/docs/configuration/storage/raft.mdx

```diff
@@ -52,4 +52,38 @@ set `ha_enabled = true`.
 - `node_id` `(string: "")` - The identifier for the node in the Raft cluster.
   This value can be overridden by setting the `VAULT_RAFT_NODE_ID` environment variable.
 
+- `performance_multiplier` `(integer: 0)` - An integer multiplier used by
+  servers to scale key Raft timing parameters. Omitting this value or setting it
+  to 0 uses default timing described below. Lower values are used to tighten
+  timing and increase sensitivity while higher values relax timings and reduce
+  sensitivity. Tuning this, affects the time it takes Vault to detect leader
+  failures and to perform leader elections, at the expense of requiring more
+  network and CPU resources for better performance.
+
+  By default, Vault will use a lower-performance timing that's suitable for
+  minimal Vault servers, currently equivalent to setting this to a value of 5
+  (this default may be changed in future versions of Vault, depending if the
+  target minimum server profile changes). Setting this to a value of 1 will
+  configure Raft to its highest-performance mode and is recommended for
+  production Vault servers. See the note on last contact timing for more details
+  on tuning this parameter. The maximum allowed value is 10.
+
+- `trailing_logs` `(integer: 10000)` - This controls how many log entries are
+  left in the log store on disk after a snapshot is made. This should only be
+  adjusted when followers cannot catch up to the leader due to a very large
+  snapshot size and high write throughput causing log truncation before a
+  snapshot can be fully installed. If you need to use this to recover a cluster,
+  consider reducing write throughput or the amount of data stored on Vault. The
```

Comment:
Do we happen to have instructions on reducing the write throughput? If not, I think is fine.